ros2topic/ros2topic/verb/bw.py

```diff
@@ -84,27 +84,27 @@ def add_arguments(self, parser, cli_name):
 
     def main(self, *, args):
         with DirectNode(args) as node:
-            _rostopic_bw(node.node, args.topic, window_size=args.window,
-                         use_sim_time=args.use_sim_time)
+            _rostopic_bw(node.node, args.topic, window_size=args.window)
 
 
 class ROSTopicBandwidth(object):
 
-    def __init__(self, node, window_size, use_sim_time=False):
+    def __init__(self, node, window_size):
         self.lock = threading.Lock()
         self.last_printed_tn = 0
         self.sizes = []
         self.times = []
         self.window_size = window_size
-        self._clock = node.get_clock()
-        self.use_sim_time = use_sim_time
+        self.use_sim_time = node.get_parameter('use_sim_time')
```

Comment:
If 'use_sim_time' isn't declared, get_parameter will throw an exception.   
You should check if it is declared at first and then check if `use_sim_time` is true.

Reply:
Updated.